components/script/dom/dedicatedworkerglobalscope.rs

```diff
@@ -131,6 +137,11 @@ impl QueuedTaskConversion for DedicatedWorkerScriptMsg {
         DedicatedWorkerScriptMsg::CommonWorker(worker.unwrap(), WorkerScriptMsg::Common(script_msg))
     }
 
+    fn inactive_msg() -> Self {
+        // Inactive is only relevant in the context of a browsing-context event-loop.
+        unreachable!("Workers should never receive messages marked as inactive");
```

Comment:
Nit: make that `panic`, not `unreachable`, the latter is for self-contained code for which you can locally check that the invariant holds.